public/video-ui/src/util/setupStore.ts

```diff
@@ -1,13 +1,47 @@
 import { browserHistory } from 'react-router';
-import { routerMiddleware } from 'react-router-redux';
+import { routerMiddleware, routerReducer } from 'react-router-redux';
 import { storeMiddleware } from './storeMiddleware';
-
-import reducers from '../reducers/reducers';
 import { configureStore } from '@reduxjs/toolkit';
+import config from '../slices/config';
+import error from '../slices/error';
+import video from '../slices/video';
+import videos from '../slices/videos';
+import saveState from '../slices/saveState';
+import search from '../slices/search';
+import youtube from '../slices/youtube';
+import usage from '../slices/usage';
+import checkedFormFields from '../slices/checkedFormFields';
+import formFieldsWarning from '../slices/formFieldsWarning';
+import s3Upload from '../slices/s3Upload';
+import videoEditOpen from '../slices/editState';
+import uploads from '../slices/uploads';
+import path from '../slices/path';
+import pluto from '../slices/pluto';
+import workflow from '../slices/workflow';
+import targeting from '../slices/targeting';
 
 export function setupStore() {
   return configureStore({
-    reducer: reducers,
+    reducer: {
```

Comment:
Moving the reducers here is a small improvement not strictly related to the goals of this PR